camunda-modeler-plugin/main/bpmndt/PathFinder.js

```diff
@@ -82,7 +82,7 @@ export default class PathFinder {
    * @returns The found elements.
    */
   _findIncoming(subProcessId) {
-    return this.elementRegistry.find(element => {
+    return this.elementRegistry.filter(element => {
```

Comment:
there is mix of `filter` and `find` (line 89) - line 89 should be rewritten to make it more clear